docs/gen_user_guide.py

```diff
@@ -0,0 +1,7 @@
+"""Generate the User Guide."""
+import os
+
+os.system(
+    r"""pandoc -f markdown -t html5 -o user_guide.html docs\user_guide.md -c style
+    .css"""
+)
```

Comment:
The general idea is correct here, but there are a few issues with this.

1. You're using a backslash as a path separator which only works on Windows. If you use a forward slash it'll work anywhere
2. Your editor has broken this string, so `.css` is on its own line and Python will try to run it as a separate command
3. I don't think you need the `-c style.css` bit anyway -- there is no `style.css` file!
4. This will only work if you run the script from the parent folder. If you do `cd docs; python gen_user_guide.py` it'll fail, because the path to `user_guide.md` will now be wrong. You can get the path to the folder the current script is in with `Path(__file__).parent`. I'd do this to make sure both the input and output path are pointing at the right place
5. I'd use `subprocess.run` to invoke pandoc rather than `os.system`, which is deprecated (see https://hackernoon.com/calling-shell-commands-from-python-ossystem-vs-subprocess-mc253z4f). You'll want the `check=True` option enabled